databricks_cli/dbfs/api.py

```diff
@@ -45,29 +45,39 @@ class ParseException(Exception):
 
 
 class FileInfo(object):
-    def __init__(self, dbfs_path, is_dir, file_size):
+    def __init__(self, dbfs_path, is_dir, file_size, modification_time):
         self.dbfs_path = dbfs_path
         self.is_dir = is_dir
         self.file_size = file_size
+        self.modification_time = modification_time
 
     def to_row(self, is_long_form, is_absolute):
         path = self.dbfs_path.absolute_path if is_absolute else self.dbfs_path.basename
         stylized_path = click.style(path, 'cyan') if self.is_dir else path
         if is_long_form:
             filetype = 'dir' if self.is_dir else 'file'
-            return [filetype, self.file_size, stylized_path]
+            # Do not return modification time if it is not available from server.
+            if self.modification_time is None:
+                return [filetype, self.file_size, stylized_path]
+            else:
+                return [filetype, self.file_size, stylized_path, self.modification_time]
         return [stylized_path]
 
     @classmethod
     def from_json(cls, json):
         dbfs_path = DbfsPath.from_api_path(json['path'])
-        return cls(dbfs_path, json['is_dir'], json['file_size'])
+        # If JSON doesn't include modification_time data, replace it with None.
+        if 'modification_time' not in json:
+            return cls(dbfs_path, json['is_dir'], json['file_size'], None)
+        else:
+            return cls(dbfs_path, json['is_dir'], json['file_size'], json['modification_time'])
```

Comment:
nit: Another way in which you could do this is the following:
```
modification_time = json['modification_time'] if `modification_time` in json else None
return cls(dbfs_path, json['is_dir'], json['file_size'], modification_time)
```
Similar to the other comment, I don't have a strong opinion about it, it's just a matter of style.

Reply:
I prefer the way you have shown as well but a similar example right below these were written with the current style. However, having a second look to the file both styles seem to be used. I will apply these changes to avoid code duplication.